### ISSUE TITLE
Cosmic File Compilation

### ISSUE THREAD

**Issue by nater1983** [pop-os/cosmic-files#392]
![image](https://github.com/user-attachments/assets/4ee0f309-9e13-414a-a0f3-97fb9ee81c6b)

This is the error I am getting well trying to compile Cosmic Files git commit aebc65200685517fff0785cf17636d92218d7ba6 .

**Comment by AngryPenguinPL** [comment]
Same problem here on OpenMandriva in cosmic-file and in cosmic-edit. Some packages compile with just without any problems and some not. I use last alpha tag.

Any idea?

**Comment by nater1983** [comment]
I have add this to my Slackbuilds
`
export VERGEN_GIT_COMMIT_DATE="$(date --utc '+%Y-%m-%d %H:%M:%S %z')"
export VERGEN_GIT_SHA=$_commit`

**Comment by AngryPenguinPL** [comment]
indeed, this allowed me to compile the package without a problem. Thanks

**Comment by KodeBarista** [comment]
This issue appears to be fixed. Compilation works both from a Git checkout and from the release source archive.
